Dev4Agriculture.ISO11783.ISOXML/ISOXML.cs

```diff
@@ -273,6 +275,31 @@ public static ISOXML Load(string path, bool loadBinData = true)
             return isoxml;
         }
 
+        /// <summary>
+        /// Load an ISOXML TaskSet and return an ISOXML Object
+        /// </summary>
+        /// <param name="stream">zip file stream</param>
+        /// <param name="loadBinData">Shall all binary data such as grids and TLGs be loaded? Default is true</param>
+        /// <returns></returns>
+        public static ISOXML Load(Stream stream, bool loadBinData = true)
+        {
+            var path = Path.GetTempPath();
+            using (ZipArchive archive = new ZipArchive(stream, ZipArchiveMode.Read))
+            {
```

Comment:
Any reason we should stick to the explicit type? Otherwise please change to var.